onert-micro/tests/mbed-os/CMakeLists.txt

```diff
@@ -0,0 +1,145 @@
+SET(CMAKE_ASM_FLAGS "-g3 -x assembler-with-cpp -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -fmessage-length=0 -fexceptions -ffunction-sections -fdata-sections -funsigned-char -MMD -fomit-frame-pointer -Og -DMBED_DEBUG -DMBED_TRAP_ERRORS_ENABLED=1 -DMBED_MINIMAL_PRINTF -mcpu=cortex-m7 -mthumb -mfpu=fpv5-d16 -mfloat-abi=softfp  -include ${CMAKE_CURRENT_SOURCE_DIR}/mbed_config.h")
+SET(CMAKE_CXX_LINK_FLAGS "-Wl,--gc-sections -Wl,--wrap,main -Wl,--wrap,_malloc_r -Wl,--wrap,_free_r -Wl,--wrap,_realloc_r -Wl,--wrap,_memalign_r -Wl,--wrap,_calloc_r -Wl,--wrap,exit -Wl,--wrap,atexit -Wl,-n -Wl,--wrap,printf -Wl,--wrap,sprintf -Wl,--wrap,snprintf -Wl,--wrap,vprintf -Wl,--wrap,vsprintf -Wl,--wrap,vsnprintf -Wl,--wrap,fprintf -Wl,--wrap,vfprintf -mcpu=cortex-m7 -mthumb -mfpu=fpv5-d16 -mfloat-abi=softfp -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -fmessage-length=0 -fexceptions -ffunction-sections -fdata-sections -funsigned-char -MMD -fomit-frame-pointer -Og -DMBED_DEBUG -DMBED_TRAP_ERRORS_ENABLED=1 -DMBED_MINIMAL_PRINTF -mcpu=cortex-m7 -mthumb -mfpu=fpv5-d16 -mfloat-abi=softfp -DMBED_ROM_START=0x8000000 -DMBED_ROM_SIZE=0x200000 -DMBED_RAM_START=0x20000000 -DMBED_RAM_SIZE=0x20000 -DMBED_RAM1_START=0x24000000 -DMBED_RAM1_SIZE=0x80000 -DMBED_BOOT_STACK_SIZE=1024 -DXIP_ENABLE=0 ")
+SET(CMAKE_CXX_LINK_FLAGS "${CMAKE_CXX_LINK_FLAGS} ${LD_SYS_LIBS} -T ${CMAKE_BINARY_DIR}/benchmark_pp.link_script.ld")
+
+ADD_DEFINITIONS(
+        -DARM_MATH_CM7
+        -DCOMPONENT_FLASHIAP=1
+        -DDEVICE_ANALOGIN=1
+        -DDEVICE_ANALOGOUT=1
+        -DDEVICE_CAN=1
+        -DDEVICE_CRC=1
+        -DDEVICE_EMAC=1
+        -DDEVICE_FLASH=1
+        -DDEVICE_I2C=1
+        -DDEVICE_I2CSLAVE=1
+        -DDEVICE_I2C_ASYNCH=1
+        -DDEVICE_INTERRUPTIN=1
+        -DDEVICE_LPTICKER=1
+        -DDEVICE_MPU=1
+        -DDEVICE_PORTIN=1
+        -DDEVICE_PORTINOUT=1
+        -DDEVICE_PORTOUT=1
+        -DDEVICE_PWMOUT=1
+        -DDEVICE_RESET_REASON=1
+        -DDEVICE_RTC=1
+        -DDEVICE_SERIAL=1
+        -DDEVICE_SERIAL_FC=1
+        -DDEVICE_SLEEP=1
+        -DDEVICE_SPI=1
+        -DDEVICE_SPISLAVE=1
+        -DDEVICE_SPI_ASYNCH=1
+        -DDEVICE_STDIO_MESSAGES=1
+        -DDEVICE_TRNG=1
+        -DDEVICE_USBDEVICE=1
+        -DDEVICE_USTICKER=1
+        -DDEVICE_WATCHDOG=1
+        -DEXTRA_IDLE_STACK_REQUIRED
+        -DMBED_BUILD_TIMESTAMP=1640167847.81
+        -DMBED_TICKLESS
+        -DSTM32H743xx
+        -DTARGET_CORTEX
+        -DTARGET_CORTEX_M
+        -DTARGET_FF_ARDUINO_UNO
+        -DTARGET_LIKE_CORTEX_M7
+        -DTARGET_LIKE_MBED
+        -DTARGET_M7
+        -DTARGET_MCU_STM32
+        -DTARGET_MCU_STM32H7
+        -DTARGET_MCU_STM32H743xI
+        -DTARGET_NAME=NUCLEO_H743ZI2
+        -DTARGET_NUCLEO_H743ZI2
```

Comment:
Mbed will always be build for this NUCLEO_H743ZI2? And if we want to introduce new board, we should add new CMakeLists.txt? Maybe create another folder for concrete board, like `onert-micro/tests/mbed-os/NUCLEO_H743ZI2/CMakeLists.txt`?